### ISSUE TITLE
Access token and file URL args are not being read from .env file, .figmaircrc or --token & --url cli parameters

### ISSUE THREAD

**Issue by poindexterous** [mikaelvesavuori/figmagic#88]
Hi all, 
I'm scratching my head a bit, I think I'm following the documentation but I can't seem to get my access token or my file ID variables to be pulled into figmagic as I try to run it for the first time. I've used postman to verify that the file ID and token work when pinging the figma API. I'm wondering if I'm just missing something silly and not seeing it, or if it might be a bug or some sort of system compatibility issue. Thanks in advance for any help and pointers. Here's what I've tried so far:

**Describe the bug**
FigmaToken & FigmaUrl variables aren't read from .env, .figmaicrc or CLI --token or--url command parameters

**To Reproduce**
1. Navigate project folder where figmagic will be installed
2. create .env file with 'FIGMA_URL' and 'FIGMA_TOKEN' entries
3. run 'npm install figmagic -D' to install locally (I do not want a global install for this use case)
4. add '"figmagic": "node ./node_modules/figmagic/build/index.js"' to scripts section of package.json
5. run ' npm run figmagic'
6. see error "Error: Error: Error: Missing one or more of required arguments: "figmaToken", "figmaUrl" when attempting to get data from Figma API!"

Also tried:
1. run "npm figmagic init"
2. added token & url entries to figmaicrc file
3.  run ' npm run figmagic'
4. see error "Error: Error: Error: Missing one or more of required arguments: "figmaToken", "figmaUrl" when attempting to get data from Figma API!"

Also tried:
1. run "npm run figmagic --token {figma-access-token-here} --url {file-id-here}"
2. see error "Error: Error: Error: Missing one or more of required arguments: "figmaToken", "figmaUrl" when attempting to get data from Figma API!"

**Expected behavior**
Expected figmagic to read and parse my figma file.

**Screenshots**
Example .figmaicrc
![image](https://user-images.githubusercontent.com/26029453/106329054-6cf28d00-6246-11eb-99e8-3e07199d489a.png)

Example .env
![image](https://user-images.githubusercontent.com/26029453/106329203-aaefb100-6246-11eb-886c-9f77115876b0.png)

Attempts to use the --url and --token params
![image](https://user-images.githubusercontent.com/26029453/106328837-17b67b80-6246-11eb-86cd-ef1dd4825dfb.png)

Checking token and file ID in postman pinging the Figma API to confirm that the token and file ID are good
![image](https://user-images.githubusercontent.com/26029453/106329715-919b3480-6247-11eb-999d-73b6952b73de.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Figmagic version 6.13.4
 - node version 12.14.0
 - using powershell as my terminal

**Additional context**
trying to install and run locally inside the project folder as opposed to globally as it's better for our use case. Checked both the token and the file ID in postman pinging the figma API to make sure they were working. I'm able to receive a valid response in postman using those credentials, I also tried a new blank starter file from https://www.figma.com/community/file/821094451476848226 just in case I had made a change that was preventing the file from being parsed in Figmagic. I'm just using the figma file ID and not the entire URL as the file id value, but I also tried using the path just to see if I'd get a different result (I did not get a different result).

**Comment by mikaelvesavuori** [comment]
Hey, thanks for reaching out!

I created a local demo scenario with a blank folder, ran `npm init -y`, `npm install figmagic -D`, added the `figmagic` command to the scripts section, and then ran various variants of files being present/absent.

What I have not done at this time is try this on a Windows setup (I develop on Mac, but can probably squeeze in some time to try this on my PC during the weekend). It may be possible that Windows has some differences in file reading/handling that are not covered.

Figmagic is built/tested for Node 14, which could be something to look at, though I doubt it.

## Things I did

- Your attempt at running `npm run figmagic --token {token} --url {url}` will not work since you use a local install. You should try `node ./node_modules/figmagic/build/index.js --token {token} --url {url}` which will work just fine (this is also documented in the Figmagic example repo's `package.json`; obscure but true).
- I released an update (4.0.6) which takes care of the issue you experienced, where token/URL will not be accepted from `.figmagicrc`; I have also verified that functionality by using a `.figmagicrc` file with token and URL values. I accidentally forgot a console log that outputs the configuration object. That might be helpful to try? I will however put out yet another version (4.0.7) that removes it.

## Actions

- Verify that you are using the latest version (4.0.7). There is no `6.13.4` version.
- Verify your file names. It might be just "fat fingers" when you typed your issue, but ensure the files are called `.figmagicrc` and/or `.env` respectively.
- Note: Your `.env` and/or `.figmagicrc` files should be stored in the root of your project, _not_ in the Figmagic dependency folder.

## Errors that _should_ be thrown

Reflection: The error you see is just before getting data with the API. Given there is no URL/token, things should break before you get near that. It's as if you had an older version...?

### No valid keys found

```
Error: Error: Could not retrieve any data. Are you missing a valid API key?
```

### Wrong URL

```
Writing design tokens...
Error: Error: TypeError: Cannot read property 'children' of undefined
```

### Wrong token

```
Attempting to fetch data from Figma API...
Error: Error: Could not retrieve any data. Are you missing a valid API key?
```

**Comment by mikaelvesavuori** [comment]
Also, might sound dumb and may not be anything, but do test your `.env` file with the kind-of truncated format, like so:
```
FIGMA_URL={url}
FIGMA_TOKEN={token}
```
instead of using spaces (`FIGMA_URL = {url}`).

Just in case there is something dumb going on under the hood.

**Comment by mikaelvesavuori** [comment]
I verified that I get this working on Windows 10 with both VS Code and in Terminal and in PS7 using the above.

**Comment by poindexterous** [comment]
Thanks @mikaelvesavuori!!! I edited the .env file to get rid of the spaces and everything worked. I feel pretty sheepish but I figured it was either something silly or I'd stumbled across something temperamental in windows. Glad it was the former. Thanks for taking the time to investigate even though it turned out to be user error on my end.  As far as versions go, it turns out that I had figmagic version 4.0.5, so I went ahead and updated to 4.0.6 which also worked just fine (I like the verbose output- very helpful). 

I'm pretty excited to get started now that everything is working! Thanks again for your help.
